build.rs

```diff
@@ -63,6 +64,8 @@ fn main() -> std::io::Result<()> {
         .type_attribute("WalletBalanceResponse", "#[derive(serde::Deserialize, serde::Serialize)]")
         .type_attribute("AccountBalanceEntry", "#[derive(serde::Deserialize, serde::Serialize)]")
         .type_attribute("WalletAccountBalance", "#[derive(serde::Deserialize, serde::Serialize)]")
+        //WalletUnlocker fields
+        .type_attribute("GenSeedResponse", "#[derive(serde::Deserialize, serde::Serialize)]")
```

Comment:
May want to add in `UnlockWalletRequest` and `UnlockWalletResponse`

Reply:
> May want to add in `UnlockWalletRequest` and `UnlockWalletResponse`

Went to add these and it gets to `FundingTxid` which is not in the proto files in which can't inherit serde.